### PR TITLE
fixed: fix some condition that was missed in eclipse io update

### PR DESCRIPTION
**Pull request by akva2**
woops...

### CODE REVIEW

**Comment by akva2** [comment]
polite ping, this is the last thing holding back restored nightly builds.

**Comment by atgeirr** [comment]
Let's Jenkins it and merge.

**Comment by atgeirr** [comment]
jenkins build this please

File: CMakeLists.txt [hunk 1]
```diff
@@ -131,9 +131,9 @@ add_custom_target(ewoms_prepare)
 EwomsAddApplication(ebos
                     SOURCES ebos/ebos.cc
                     EXE_NAME ebos
-                    CONDITION OPM_GRID_FOUND AND OPM_PARSER_FOUND)
+                    CONDITION OPM_GRID_FOUND AND HAVE_ECL_INPUT AND HAVE_ECL_OUTPUT)
```
Comment (andlaus) [inline CMakeLists.txt]: small question out of curiosity: is it better to use `ENABLE_ECL_{IN,OUT}PUT` or `HAVE_ECL_{IN,OUT}PUT`? is there a difference?

Comment (akva2) [reply CMakeLists.txt]: ENABLE_XXXX are options in opm-common build system only. HAVE_XXX is the variables passed to downstreams through the config file. so we should use HAVE_XXX.

**Comment by akva2** [comment]
not sure what that failure is about, is current state unstable in master ?

**Comment by atgeirr** [comment]
> not sure what that failure is about, is current state unstable in master ?

Looks like it, I thought it was due to the missing merge of OPM/opm-simulators#1427, but apparently not.

**Comment by atgeirr** [comment]
Kai is fixing this, as it is the same problem as in OPM/opm-simulators#1427. I do not quite understand why yesterday's Jenkins test did not pick this up though.

**Comment by GitPaean** [comment]
I think I saw `test_ecl_output` failure in https://github.com/OPM/opm-parser/pull/1223, while somehow we did not manage to see it again in https://github.com/OPM/opm-common/pull/342

**Comment by akva2** [comment]
that's easily explained. this PR re-enables the building of the test.

**Comment by atgeirr** [comment]
> that's easily explained. this PR re-enables the building of the test.

Thanks for the explanation. That also means that Jenkins is not broken...

**Comment by GitPaean** [comment]
jenkins build this ewoms=297 please

**Comment by GitPaean** [comment]
Sorry, it still fails. I am updating the PR. 

BTW: what is the option to enable the testing building in ewoms? so I can test locally.

**Comment by akva2** [comment]
-DBUILD_TESTING=1 to build everything by default, or 'make test-suite' to build test without enabling them by default, or simply 'make test_ecl_output' to only build the one test.

**Comment by GitPaean** [comment]
Oh, sorry. I did not realize this PR is also in ewoms. So we could not test these two PR together.

**Comment by GitPaean** [comment]
I created a pull request to make PR #297 to join this PR.

**Comment by GitPaean** [comment]
> -DBUILD_TESTING=1 to build everything by default, or 'make test-suite' to build tests without enabling them by default, or simply 'make test_ecl_output' to only build the one test.

It did not work on my local machine. I am not sure the reason is.

**Comment by akva2** [comment]
again, it's because you are missing this pr - the test is disabled due to broken conditions in master.

**Comment by akva2** [comment]
jenkins build this please

**Comment by andlaus** [comment]
jenkins is green. merging.

**Pull request merged**